Detectors/MUON/MCH/Simulation/src/Digitizer.cxx

```diff
@@ -152,14 +152,18 @@ int Digitizer::processHit(const Hit& hit, int detID, double event_time)
     auto ymin = (localY - seg.padPositionY(padid)) - dy;
     auto ymax = ymin + dy;
     auto q = resp.chargePadfraction(xmin, xmax, ymin, ymax);
-    if (seg.isBendingPad(padid)) {
-      q *= chargebend;
-    } else {
-      q *= chargenon;
+    if (resp.aboveThreshold(q)) {
```

Comment:
do I understand correctly that the threshold has to be itself "fractional" ?

Reply:
@aphecetche 
yes, this is the threshold in aliroot.
I am thinking of multipliying by the the inverse threshold afterwards to keep all significant digits, since I don't keep a charge separately in the digit object.